tools/c7n_azure/c7n_azure/resources/postgresql_database.py

```diff
@@ -0,0 +1,60 @@
+# 2019 Microsoft Corporation
```

Comment:
`Copyright 2019 Microsoft Corporation`

I suspect the word `Copyright` is the most important part :)